frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/utils/optionsUtils.test.ts

```diff
@@ -1,120 +1,238 @@
 import { SelectedOptionsType } from '../../../../../../components/config/editModal/EditOptions/EditOptions';
-import type { IOption } from '../../../../../../types/global';
+import type { Option } from 'app-shared/types/Option';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import type { FormItem } from '../../../../../../types/FormItem';
+import type { SelectionComponentType } from '../../../../../../types/FormComponent';
 import {
   getSelectedOptionsType,
   getSelectedOptionsTypeWithManualSupport,
   componentUsesDynamicCodeList,
+  hasOptionListChanged,
+  handleOptionsChange,
+  resetComponentOptions,
+  updateComponentOptionsId,
+  updateComponentOptions,
+  isOptionsIdReferenceId,
+  isOptionsModifiable,
 } from './optionsUtils';
+import { componentMocks } from '../../../../../../testing/componentMocks';
 
-describe('getSelectedOptionsType', () => {
-  it('should return SelectedOptionsType.Unknown if both options and optionsId are set', () => {
-    const codeListId = 'codeListId';
-    const options: IOption[] = [{ label: 'label1', value: 'value1' }];
-    const optionListIds = ['codeListId'];
-    const result = getSelectedOptionsType(codeListId, options, optionListIds);
-    expect(result).toEqual(SelectedOptionsType.Unknown);
-  });
+// Test data:
+const mockedComponent: FormItem<SelectionComponentType> =
+  componentMocks[ComponentType.RadioButtons];
+const optionListIds = ['some-id'];
```

Comment:
should this string be a const so we can reference it directly in the tests that uses it?

Reply:
Since all test cases in this file uses its own local data, maybe this one should be local in the `IsOptionsIdReferenceId` tests as well? 😊 
But still use the variable reference in the optionListIds so the same variable can be used in the tests.

Reply:
With this comment I meant to define the optionListIds per test case as we do in the rest of the tests in this file 🫣 
Then it will be easier to see the purpose of each test isolated